src/main/java/at/hannibal2/skyhanni/config/storage/ProfileSpecificStorage.java

```diff
@@ -769,7 +781,52 @@ public static class RabbitData {
         @Expose
         public long millisInCf = 0;
```

Comment:
this should be changed to be a Duration instead of a long, but thats out of scope of this pr (since i think you would have to make a wrapper for it). if you could add a todo here for that it would be great